apps/judicial-system/web/src/routes/Shared/Statement/Statement.tsx

```diff
@@ -72,63 +73,44 @@ const Statement = () => {
       : constants.SIGNED_VERDICT_OVERVIEW_ROUTE
   }/${id}`
 
-  const newUploadFiles = uploadFiles.filter((file) => {
-    return (
-      workingCase.caseFiles?.find((caseFile) => caseFile.id === file.id) ===
-      undefined
```

Comment:
Same as above. A slight change in the way we handle individual file states make this unnecessry now. When uploading files, it is now enough to locate all files with upload percent 0.